style/article.less

```diff
@@ -121,6 +121,11 @@
 						background-color: #eaecf0;
 						margin-top: -10px;
 					}
+
+					&[ data-selected='true' ] {
```

Comment:
Is it possible that we're starting to have those duplicated quite a bit? If so, do you think they can be extracted to variables or mixins?

Reply:
I think we can set the global mixin for styling when `data-selected` is true, can do some customization when needed (for example figure doesn't need border)

Reply:
I have made this as variables, there are too many variations between text, image, card, so I'm not using mixin here.